src/toil_scripts/adam_gatk_pipeline/align_and_call.py

```diff
@@ -148,6 +148,11 @@ def build_parser():
                         help = "Skip preprocessing and start running from variant calling. Implies --skip_alignment.",
                         action = 'store_true')
 
+    # what directory are our sequence files in?
+    parser.add_argument('-SD', '--sequence_dir',
```

Comment:
Unix convention is that `-SD` is the contraction of `-S -D`. I would either use --sd or ditch it completely.